src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I don't quite understand why we need this. manageGroups seems to vague to be part of an API service. The name of the operation should be very clear about the purpose of what is trying to achieve.